modules/ROOT/Edits/edifact-receive-read-settings.adoc

```diff
@@ -0,0 +1,100 @@
+= Configure Receive Settings for Inbound EDIFACT messages
+
+Partner Manager receive settings include the EDIFACT acknowledgments and validation rules to apply when processing an inbound EDIFACT message from the specified partner.
+
+To configure EDIFACT receive settings:
+
+. In the navigation menu, select *Partners*.
+. Select the partner to which these settings apply.
+. In the *Validation and acknowledgment settings* section's *Receive from <partner-name>* column, click *EDIFACT*.
+. Select a link to configure:
+* <<ack-settings,Acknowledgment settings>>
+* <<validation-settings,Validation settings>>
+* <<control-numbers,Control Numbers>>
+. Click *Save*.
+
+[NOTE]
+It can take up to five minutes or so for your configuration changes to sync with Mule.
+
+[[ack-settings]]
+== Acknowledgment Settings
+
+Optionally configure acknowledgment settings for each Inbound EDIFACT message flow. To configure acknowledgement settings:
+
+* Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
++
+When there is a routing failure, this acknowledgement indicates that the message is rejected.
+
+If you enable one of these options, you must either select the partner endpoint to which to send the acknowledgements or create a new endpoint for this purpose.
+
+== Validation Settings
+
+Select one or more of the following options to ensure that all EDIFACT messages received from the trading partner meet validation rules:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default
```

Comment:
We could omit the Default column and simply end the description of the default description text with "(default)"....

Reply:
Also, are we using "unselected" instead of "not selected" because we are referring to taking the action of removing (default?) selection, or do we really mean just the state of not being selected? Probably too granular to bother the audience, but, well, you know...word nerd....